www/js/shortcuts/shortcut-functions.js

```diff
@@ -0,0 +1,166 @@
+const electron = require('electron');
```

Comment:
can we name the folder `keyboard-shortcuts`?

Reply:
sure